scripts/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+
+if [ -f ${JAVA_HOME}/bin/java ]
+then
+    JAVA_BIN=${JAVA_HOME}/bin/java
+elif [ -f /usr/bin/java ]
+then
+    JAVA_BIN=/usr/bin/java
+fi
+
+if [ -z "$JAVA_BIN" ]
+then
+    echo "no java executable found. Please set your JAVA_HOME"
+fi
+
+
+JAR_FILE=$(mktemp)
+
+curl -s "https://dl.bintray.com/dailymotion/com.dailymotion.kinta/com/dailymotion/kinta/kinta-cli/0.1.0/kinta-0.1.0.jar" > $JAR_FILE
```

Comment:
This should not be something like : 
https://dl.bintray.com/dailymotion/com.dailymotion.kinta/com/dailymotion/kinta/kinta-cli/$v/kinta-cli-$v.jar
?

Reply:
I wish but it's not easy to get the latest version from shell unfortunately

Reply:
This should be better now that I deploy the last zip to https://dailymotion.github.io/kinta/kinta.zip